tests/e2e/core-tests/specs/merchant/wp-admin-order-apply-coupon.test.js

```diff
@@ -1,43 +1,35 @@
-/* eslint-disable jest/no-export, jest/no-standalone-expect */
-
 /**
  * Internal dependencies
  */
 const {
 	createSimpleProduct,
-	createSimpleOrder,
 	createCoupon,
 	uiUnblocked,
-	addProductToOrder,
 	evalAndClick,
-	merchant
+	merchant,
+	createOrder,
 } = require( '@woocommerce/e2e-utils' );
 
 const config = require( 'config' );
-const simpleProductName = config.get( 'products.simple.name' );
 const simpleProductPrice = config.has('products.simple.price') ? config.get('products.simple.price') : '9.99';
 const discountedPrice = simpleProductPrice - 5.00;
 
 const couponDialogMessage = 'Enter a coupon code to apply. Discounts are applied to line totals, before taxes.';
 
 let couponCode;
 let orderId;
+let productId;
 
 const runOrderApplyCouponTest = () => {
 	describe('WooCommerce Orders > Apply coupon', () => {
 		beforeAll(async () => {
-			await createSimpleProduct();
+			productId = await createSimpleProduct();
 			couponCode = await createCoupon();
-			
-			await merchant.login();
-			orderId = await createSimpleOrder('Pending payment', simpleProductName);
-			
-			await Promise.all([
-				addProductToOrder(orderId, simpleProductName),
+			orderId = await createOrder( { productId: productId, status: 'pending' } );
```

Comment:
`productId: productId,` can just be `productId,`.

Reply:
Good catch fixed!